docs/package.json

```diff
@@ -23,7 +23,7 @@
     "prod:deploy": "node scripts/deploy/aws.js",
     "prettier-all": "npx prettier --write '{src,static-config-helpers,static-config-parts,static.config,scripts,lib}/**/*.{js,jsx}'",
```

Comment:
GLOBAL: Can we globally search and get rid of all `npx` usage in our `package.json:scripts`? It's slow and out of sync with our actual dependencies.

Reply:
I think this might be a regression, I had thought we got rid of this earlier. Either way, will get rid of it 👍